wearable-samples/ReferenceApplication/WApps/RotarySelector/RotarySelectorItem.cs

```diff
@@ -39,6 +39,19 @@ public class RotarySelectorItem : ImageView
         private RotaryBadege deleteBadge;
         public bool IsDeleteAble { get; set; }
 
+        private float itemOpacity = 1.0f;
+        public float ItemOpacity
+        {
+            get
+            {
+                return itemOpacity;
+            }
+            set
+            {
+                itemOpacity = value;

```

Comment:
`+                Opacity = itemOpacity;`

And after modifying test app it only then works:

```
-            float opacity = last.Opacity != 0.5f ? 0.5f : 1.0f;
-            last.Opacity = opacity;
+            float opacity = last.ItemOpacity != 0.5f ? 0.5f : 1.0f;^M
+            last.ItemOpacity = opacity;^M

```